packages/commonjs-shim/src/type.ts

```diff
@@ -0,0 +1,6 @@
+import type { SourceMapInput } from 'rollup';
```

Comment:
please rename to `types.ts` to match the pattern in the repo

Reply:
done.